### PR TITLE
Update ActionText docs [ci skip]

### PR DESCRIPTION
**Pull request by jackkinsella**
- Added guidance on installation with `webpacker`. I always like having a reference as to how things should be connected rather than depending on installers, which often don't work in projects with non-standard structures.
- Clarify confusion about whether `content` attribute in database is needed.

### CODE REVIEW

File: guides/source/action_text_overview.md [hunk 1]
```diff
@@ -45,9 +45,31 @@ happens after every keystroke, and avoids the need to use execCommand at all.
 
 ## Installation
 
-Run `rails action_text:install` to add the Yarn package and copy over the necessary migration.
-Also, you need to set up Active Storage for embedded images and other attachments.
-Please refer to the [Active Storage Overview](active_storage_overview.html) guide.
+Run `rails action_text:install` to add the Yarn package and copy over the necessary migration. Also, you need to set up Active Storage for embedded images and other attachments. Please refer to the [Active Storage Overview](active_storage_overview.html) guide.
+
+### Installation with Webpacker
```
Comment (rafaelfranca) [inline guides/source/action_text_overview.md]: Now those instructions will be confusion for people that have run the installer and it was successful. They will not know if they should follow those steps or not. Can you clarify?

**Review by rafaelfranca — CHANGES_REQUESTED** [review]

File: guides/source/action_text_overview.md (hunk 1 above)
Comment (jackkinsella) [reply guides/source/action_text_overview.md]: I see your point. Zooming out, the issue I'm trying to solve for is when the installer fails, as happened in my case (I store my webpacker stuff in `app/webpacker`). I'd like for the docs to have a fallback reference.

Perhaps I could relabel the new section "Troubleshooting the Installation"?

Comment (rafaelfranca) [reply guides/source/action_text_overview.md]: I imagine that was the problem. We try to focus on explaining what the installer will do, and maybe avoid the new subsection.

Something like:

> Run `rails action_text:install` to add the Yarn package and copy over the necessary migration. The installer will...

Comment (jackkinsella) [reply guides/source/action_text_overview.md]: OK I've reframed in terms of what happen as you suggested.

**Pull request merged**